src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -275,24 +278,23 @@ public PKAnalysis CreatePKAnalysisFromValues(PKValues pkValues, Simulation simul
          return _pkMapper.MapFrom(compound.MolWeight, pkValues, options.PKParameterMode, compound.Name);
       }
 
-      private string lowerSuffix(string text)
+      /// <summary>
+      /// Returns the range strings when the <paramref name="text"/> contains 'Range 2.5% to 97.5%' language
+      /// </summary>
+      /// <param name="text">The text being split</param>
+      /// <returns>The individual range descriptions as an array with index 0 containing low range and index 1 containing high range.
+      /// If the string cannot be split on 'Range', returns the original text in both indices of the array</returns>
+      private string[] rangeDescriptions(string text)
```

Comment:
and here you say
`private (string lowerRange, string upperRange) = rangeDescription...`